parse_tournament.py

```diff
@@ -1,45 +1,48 @@
 import mysql.connector
 import json
-from urllib2 import urlopen
-from dateutil import parser as timeparser
+import requests
 
 def prompt(type, value):
-	return raw_input("Retrieved "+type+": "+value+". Press Enter if this is correct or type the correct value:\n\t") or value
+	return input("Retrieved "+type+": "+value+". Press Enter if this is correct or type the correct value:\n\t") or value
 
 db_connection = mysql.connector.connect(
 		user='root',
-		password='paradise',#'chunkylove69@aol.com',
+		password='root',
 		host='localhost',
 		database='MeleeData'
 	)
 db_cursor = db_connection.cursor()
 
 while True:
 	entry = {}
-	host = raw_input("Enter a bracket host, eg. smash.gg or challonge.com (q to exit):\n\t")
+	host = input("Enter a bracket host, eg. smash.gg or challonge.com (q to exit):\n\t")
 	if host == 'q':
 		break
 
 	if host == 'c' or host == 'challonge' or host == 'challonge.com':
 		entry['host'] = 'challonge'
 
-		entry['id_string'] = raw_input("Enter an id string, eg. michigansmash-umeme54:\n\t")
+		entry['id_string'] = input("Enter an id string, eg. michigansmash-umeme54:\n\t")
 		if entry['id_string'] == 'q':
 			continue
 
 		data = {}
+		participants = {}
 		try:
-			for data_type, subpath in [(t, entry['id_string']+s) for t,s in [('tournament',''),('matches','/matches')]]:
+			for data_type, subpath in [(t, entry['id_string']+s) for t,s in [('tournament',''),('matches','/matches'),('participants','/participants')]]:
 				uri = 'https://api.challonge.com/v1/tournaments/'+subpath+'.json'
 				print("Contacting API at %s..." % uri)
-				data[data_type] = json.loads(urlopen(uri+'?api_key=XBFwcbaWSvrfHiaNONNgwyfPo8LrYozALwIWfkBd').read())
+				response = requests.get(uri+'?api_key=XBFwcbaWSvrfHiaNONNgwyfPo8LrYozALwIWfkBd')
+				data[data_type] = response.json()
 		except Exception as e:
-			print(e)
 			print("Error accessing %s." % uri)
+			print(e)
 			continue
 		data['tournament'] = data['tournament']['tournament']
 		entry['name'] = prompt('name', data['tournament']['name'])
-		entry['date'] = prompt('date', timeparser.parse(data['tournament']['started_at']).date())
+		entry['date'] = prompt('date', data['tournament']['started_at'])
+		entry['date'] = entry['date'].split('T')[0] + " " + entry['date'].split('T')[1]
+		entry['date'] = entry['date'][:-6] 
```

Comment:
I assume you took out the library because it's not python3, so that's fine, but we should find a replacement for the future instead of manually parsing the string. Doesn't have to be right now, just keep in mind.